docs/src/tutorials/creating_geometry.md

```diff
@@ -51,7 +51,7 @@ plot!(ax, multipoint; marker = '☁', markersize = 30)
 fig
 ````
 
-Let's create a `Line` between two points.
+Let's create a `LineString` connecting two points.
```

Comment:
So here I thought it would be most intuitive to start with a `Line` between two points then to move next to `LineString`, like we did from `Point` to `MuliPoint`... the only reason the demo code used `LineString` is because `Line` would not plot. If a `Line` will not plot in GeoMakie then we could change the code not the text

Reply:
Let's leave it how you have it here and we can update later once `Line` can be plotted on a released version of Makie

Reply:
Line is pretty rare and a lot of C libraries don't actually support it.   IMO it's probably best not to advertise it in a user tutorial - there are a lot of sharp edges there.